src/lib/ui/ExtensionsList.svelte

```diff
@@ -1,13 +1,54 @@
 <script lang="ts">
-import type { ExtensionByCategoryInfo } from '$lib/api/extensions-info';
+import type { CatalogExtensionInfo, ExtensionByCategoryInfo } from '$lib/api/extensions-info';
 
+import ExtensionByCategoryCard from './ExtensionByCategoryCard.svelte';
 import ExtensionsByCategory from './ExtensionsByCategory.svelte';
 
-const { extensionsByCategories }: { extensionsByCategories: ExtensionByCategoryInfo[] } = $props();
+export let extensionsByCategories: ExtensionByCategoryInfo[];
+
+let filteredExtensions: CatalogExtensionInfo[] = [];
+let showCategories = true;
+
+// Get the query from the URL
+function getQueryLimit(): number | null {
```

Comment:
issue: null is not authorized (I was assuming we got a lint rule for that)
---> undefined

Reply:
woops! old habits